ouroboros-consensus/src/Ouroboros/Consensus/Ledger/Mock/UTxO.hs

```diff
@@ -49,6 +52,9 @@ import           Ouroboros.Consensus.Ledger.Mock.Address
 data Tx = Tx (Set TxIn) [TxOut]
   deriving (Show, Eq, Ord, Generic, Serialise)
 
+instance ToCBOR Tx where
```

Comment:
Hmm, yeah, it's not ideal. I'd say only add a `ToCBOR` when it's required of you by the `HashAlgorithm`. Otherwise stick to `Serialise` for now. Starting going down that rabbit-hole, but didn't seem like a good option to change right now.